modules/schema/src/test/java/org/apache/ignite/internal/schema/configuration/SchemaConfigurationConverterTest.java

```diff
@@ -96,7 +96,9 @@ public void createRegistry() throws ExecutionException, InterruptedException {
                         SchemaBuilders.column("COL2", ColumnType.DOUBLE).build(),
                         SchemaBuilders.column("A", ColumnType.INT8).build(),
                         SchemaBuilders.column("B", ColumnType.INT8).build(),
-                        SchemaBuilders.column("C", ColumnType.INT8).build()
+                        SchemaBuilders.column("C", ColumnType.INT8).build(),
+                        SchemaBuilders.column("DUR", ColumnType.duration()).build(),
+                        SchemaBuilders.column("PER", ColumnType.period()).build()
```

Comment:
could you please rewrite tests here and below in order to automatically verify all types from ColumnTypeSpec/NativeTypeSpec (it is OK to have ignore list though)?